packages/components/psammead-brand/src/index.test.jsx

```diff
@@ -165,6 +165,34 @@ describe('Brand', () => {
       expect(document.activeElement).toBe(brand);
     });
 
+    it('should let the SvgWrapper be focussed with a ref if no url prop', () => {
+      const TestComponent = () => {
+        const brandRef = useRef(null);
+
+        useEffect(() => {
+          brandRef.current?.focus();
+        });
+
+        return (
+          <Brand
+            product="Default Brand Name"
+            svgHeight={24}
+            maxWidth={280}
+            minWidth={180}
+            svg={svg}
+            backgroundColour={C_POSTBOX}
+            logoColour={C_WHITE}
+            data-brand="header"
+            ref={brandRef}
+          />
+        );
+      };
+
+      const { container } = render(<TestComponent />);
+      const brand = container.querySelector('div[tabindex="-1"]');
+      expect(document.activeElement).toBe(brand);
```

Comment:
It is worth also asserting the initial activeElement to ensure it has changed?

Reply:
Good idea, I'll add that too 👍